osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -60,8 +61,20 @@ public void UpdateBeatmap(WorkingBeatmap beatmap)
             BeatmapSetInfo beatmapSetInfo = beatmap.BeatmapSetInfo;
             BeatmapInfo beatmapInfo = beatmap.BeatmapInfo;
 
+            double bpmPreview = 60000 / beatmap.Beatmap.BeatLengthAt(beatmap.Beatmap.Metadata.PreviewTime);
+            double bpmMax = bpmPreview;
+            double bpmMin = bpmPreview;
+            foreach( ControlPoint a in beatmap.Beatmap.ControlPoints)

```

Comment:
This entire thing should probably also be encapsulated away into a method. `GetBPMRange` or something similar.